colors.module.scss

```diff
@@ -0,0 +1,20 @@
+$light-grey-color: #d6dadb;
+$light-grey-color-1: #c2c2c2;
+$light-grey-color-2: #f5f7fa;
+$red-color: #ff444f;
+$white-color: #fff;
+$black-color: #333;
+$black-color-1: #151717;
+$black-color-2: #0e0e0e;
+$dark-grey-color: #999;
+$dark-grey-color-1: #6e6e6e;
+$dark-grey-color-2: #999999;
+$dark-grey-color-3: #252525b8;
+$dark-grey-color-4: #1f1f1f;
+$dark-grey-color-5: #303536;
+$red-color-1: #ff444f;
+$red-color-2: #d43e47;
+$blue-color: #85acb0;
+$blue-color-2: #85acb0;
+$green-color-1: #556b2f;
+$green-color-2: #9ed178;
```

Comment:
I see you are using Sass variables here. We are using css variables that you can see in index.scss. Please move them there as --light-grey: .... and then read them by var(--light-grey)